### DISCUSSION TITLE
Disable auto switch chain after connect

### DISCUSSION THREAD

**Discussion by vantrung1408** [rainbow-me/rainbowkit#1859] [Q&A]
Can i disable auto switch chain function after connect wallet?

**Comment by 0xmkh** [comment]
@vantrung1408 Can you share a small reproducible example ? 

We don't auto switch to a chain after connecting a wallet unless your current chain for the connected wallet is not supported in the dApp. E.g if you only have ethereum configured in your dApp, but have polygon as a current chain we then auto switch to ethereum to make sure it's compatible with your dApp.
